### DISCUSSION TITLE
找不到指定的sdk microsoft.net.sdk

### DISCUSSION THREAD

**Discussion by zxc100** [LorisYounger/VPet#131] [Q&A]
下载最新的文件压缩包后，打开vs2022启动最新本文件项目后出现找不到指定的sdk microsoft.net.sdk，之前下载的旧版压缩包，打开项目后没有出现这个问题

**Comment by LorisYounger** [comment]
我也不清楚,感觉这个应该是系统缺东西, 听上去是微软的问题
